examples/plot_all_cameras.py

```diff
@@ -2,17 +2,18 @@
 from ctapipe.visualization import CameraDisplay
 from matplotlib import pyplot as plt
 import numpy as np
+from math import ceil
+
 
 
 if __name__ == '__main__':
 
     plt.style.use("bmh")
 
-    N = 2
-    M = 7
-    plt.figure(figsize=(15, 6))
-
     camera_names = CameraGeometry.get_known_camera_names()
+    M = 5
+    N = ceil(len(camera_names)/M)
+    plt.figure(figsize=(15, 6))
```

Comment:
how about going all the way and doing it like
```
n_tels = len(camera_names)
n_cols = np.ceil(np.sqrt(n_tels)).astype(int)
n_rows = np.ceil(n_tels / n_cols).astype(int)
```

Reply:
Sure, but I switched between cols and rows and rounded down for the number of rows. It helps get a nicer looking plot.

Reply:
Sure, it was just a suggestion to make both dimensions dynamically sized :-)